keyboards/redox/keymaps/finex/rules.mk

```diff
@@ -0,0 +1,7 @@
+RGBLIGHT_ENABLE = yes
+
+COMMAND_ENABLE = no
+
+ifndef QUANTUM_DIR
```

Comment:
Could you remove this (ifndef/include/endif) block? 

It's not needed, and is a holdover from the old build system.

Reply:
Of course!

Reply:
@drashna  Need I to create a new pull request?

Reply:
Nope, Just create a commit on the branch you created the pull request with, and push it.  That will automatically update the PR.